docs/track.rst

```diff
@@ -427,6 +429,16 @@ Example::
               ]
             }
           }
+        ],
+        "on-failure": [
+          {
+            "set": {
+              "field": "_index",
+              {% raw %}
+              "value": "failed-{{ _index }}"
```

Comment:
I think this is pretty complex to grasp. We should add a note about the `raw` / `endraw` tags below the example and we should also link to the [Elasticsearch docs](https://www.elastic.co/guide/en/elasticsearch/reference/current/handling-failure-in-pipelines.html) so users can lookup what this is actually doing.

Reply:
so are you saying to revert it, or to just add links to show what its doing better?